ui/src/components/inputs/Editor.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="editor-wrapper">
+    <div v-hotkey="keymap" class="editor-wrapper">
```

Comment:
not really sure about that hotkey, maybe at a confirm toast ? 
that mean ctrl+enter to have the toast and enter to confirm. 

Or maybe always show the modal is better to avoid miss click